benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -34,6 +34,16 @@ def time_broadcast(self):
         self.d - self.e
 
 
+class At(Benchmark):
+    def setup(self):
+        self.vals = np.random.rand(1_000_000)
+        self.idx = np.random.randint(1000, size=1_000_000)
```

Comment:
```suggestion
        self.idx = np.random.randint(1000, size=1_000_000).astype(np.intp)
```
Hadn't noticed, but I think it probably makes sense to ensure we are working with intp for the basic benchmark.